test/differencer/rulesProcessor.test.ts

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { NodeDiff } from "../../src/diffTool/jsonDiff";
+import { applyRules } from "../../src/diffTool/rulesProcessor";
+import * as chai from "chai";
+import path from "path";
+import fs from "fs-extra";
+import tmp from "tmp";
+import sinon from "sinon";
+import { ramlToolLogger } from "../../src";
+import chaiAsPromised from "chai-as-promised";
+
+/* eslint-disable @typescript-eslint/no-use-before-define*/
+
+const expect = chai.expect;
+chai.use(chaiAsPromised);
+
+const rulesPath = path.join(__dirname, "../../diffRules", "rules.json");
```

Comment:
1) Shouldn't this be `defaultRules.json`?
2) It looks like this is only used in the first `describe` block. Please move it there to avoid naming conflicts/confusion in other blocks.

Reply:
oh..this is not really used because I have defined again in the actual test which has the correct one. I will fix it anyway